Intersect.Editor/Forms/DockingElements/frmMapLayers.cs

```diff
@@ -1102,9 +1102,9 @@ private void InitLocalization()
             lblY.Text = Strings.Warping.y.ToString("");
             lblWarpDir.Text = Strings.Warping.direction.ToString("");
             cmbDirection.Items.Clear();
-            for (var i = -1; i < 4; i++)
+            for (var i = Direction.None; i < (Direction)4; i++)
```

Comment:
Same as above

Reply:
Uh, like I mentioned I think I contradicted myself in my review today. Leave as -1, but please document why it is different (because it is not obvious from the code perspective, the actual user-end might make more sense though).